#! /bin/sh /usr/share/dpatch/dpatch-run
## ast-config-dirs.dpatch by  Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: Allow the usage of small configuration files. ALA debian :)

@DPATCH@
diff -urNad freepbx-2.1.1/amp_conf/astetc/extensions.conf /tmp/dpep.Om4Oog/freepbx-2.1.1/amp_conf/astetc/extensions.conf
--- freepbx-2.1.1/amp_conf/astetc/extensions.conf	2006-05-24 01:15:20.000000000 +0300
+++ /tmp/dpep.Om4Oog/freepbx-2.1.1/amp_conf/astetc/extensions.conf	2006-08-16 12:28:11.000000000 +0300
@@ -13,8 +13,14 @@
 
 ; Customizations to this dialplan should be made in extensions_custom.conf
 ; See extensions_custom.conf.sample for an example
+; note that in future versions of this debian package
+; this file will be deprecated, and users are strongly 
+; encouraged to put custom files in extension.d/*.conf
 #include extensions_custom.conf
 
+; include other user/package configuration
+#include extensions.d/*.conf
+
 [from-trunk]							; just an alias since VoIP shouldn't be called PSTN
 include => from-pstn
 
diff -urNad freepbx-2.1.1/amp_conf/astetc/manager.conf /tmp/dpep.Om4Oog/freepbx-2.1.1/amp_conf/astetc/manager.conf
--- freepbx-2.1.1/amp_conf/astetc/manager.conf	2006-05-01 05:09:24.000000000 +0300
+++ /tmp/dpep.Om4Oog/freepbx-2.1.1/amp_conf/astetc/manager.conf	2006-08-16 12:28:11.000000000 +0300
@@ -14,4 +14,13 @@
 write = system,call,log,verbose,command,agent,user
 
 #include manager_additional.conf
+
+; Customizations to the manager configuration should be made in 
+; manager_custom.conf
+; note that in future versions of this debian package
+; this file will be deprecated, and users are strongly 
+; encouraged to put custom files in manager.d/*.conf
 #include manager_custom.conf
+
+; include other user/package configuration
+#include "manager.d/*.conf"
